lib/services.rb

```diff
@@ -12,6 +13,8 @@ def self.content_store
   end
 
   def self.email_sender
-    @email_sender ||= EmailSender::Notify.new
+    return @email_sender ||= EmailSender::Notify.new if
```

Comment:
I think it would be good if we could raise an exception here if we don't get `notify` or `pseudo` since it's most likely a mistake.